spec/factories/clearance.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  sequence :email do |n|
+    base_email = ENV['TEST_SENDER_EMAIL'] || "user#{n}@gmail.com"
+    base_email
+  end
+
+  factory :user do
+    email { generate(:email) }
+    password { 'password' }
+  end
+end
```

Comment:
I assume you were trying to fix tests when changing this from what clearance generated. I think you can change them back.

```suggestion
FactoryBot.define do
  sequence :email do |n|
    "user#{n}@example.com"
  end

  factory :user do
    email
    password { 'password' }
  end
end
```